autoapi/settings.py

```diff
@@ -10,3 +10,15 @@
 TEMPLATE_DIR = os.path.join(SITE_ROOT, "templates")
 
 API_ROOT = "autoapi"
+
+SINGLE_PAGE_LEVELS = [
+    "package",
+    "module",
+    "class",
+    "exception",
+    "function",
+    "method",
+    "attribute",
+    "property",
+    "data",
+]
```

Comment:
This is the variable used at internal level to control the depth level for the `autoapi_single_page_level`. It is working as expected, @AWhetter. Let me know if you prefer other order.